recipe/build.sh

```diff
@@ -3,21 +3,36 @@
 # Install to conda style directories
 [[ -d lib64 ]] && mv lib64 lib
 
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
 # Remove 32bit libraries
 [[ -d "compute-sanitizer/x86" ]] && rm -rf compute-sanitizer/x86/
 
 for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
     [[ $i == "conda_build.sh" ]] && continue
     [[ $i == "metadata_conda_debug.yaml" ]] && continue
+
     cp -rv $i ${PREFIX}
 
     if [[ $i == "compute-sanitizer" ]]; then
         for j in `ls $i`; do
             [[ -d $PREFIX/$i/$j ]] && continue
-            [[ $j == "compute-sanitizer" ]] && continue
 
-            patchelf --set-rpath '$ORIGIN/../lib' --force-rpath $PREFIX/$i/$j
+            if [[ $j == "compute-sanitizer" || $j == "TreeLauncherSubreaper" || $j == "TreeLauncherTargetLdPreloadHelper" ]]; then
+                echo patchelf --force-rpath --set-rpath "\$ORIGIN/../lib:\$ORIGIN/../${targetsDir}/lib" $PREFIX/$i/$j
+                patchelf --force-rpath --set-rpath "\$ORIGIN/../lib:\$ORIGIN/../${targetsDir}/lib" $PREFIX/$i/$j
```

Comment:
This assumes that binary files from the tarball are already given executable permissions, correct ? Should we rely on that assumption ?

Reply:
If there are binary files that are not executables, we may not want to patch them. Or if we do patch them, we may need to do something else

That said if there are binaries that should be executables that are missing the executable bit, that sounds like a packaging bug that occurred before this feedstock's build. That isn't to say we couldn't add a workaround here, but it would mean we would ultimately want to fix this issue upstream of conda-forge

So yes think this is ok to rely on. Though please let me know if there are other things we should consider

Reply:
Thanks! Changes pushed as suggested. Confirmed the same set of files patched.